Added MangaHost-BR (PT-BR) #49

viniciuslance opened the pull request:
Added MangaHost-BR (PT-BR) from request #49

CarlosEsco commented:
What are all the extra files?

masgritz commented:
I believe they are XML files to make the icon adaptive, but I'm not entirely sure.

CarlosEsco commented:
Extensions don't support adaptive so needs to clean up and and just have standard icons

masgritz commented:
I can fix it up later tonight if you want, seems like it has been a while since @viniciuslance has appeared on github.

viniciuslance commented:
It's been a while since I worked with this extensions, I'll try to figure out what's wrong, but must be files that android studio created

CarlosEsco commented:
Icons is messed up, apk shouldn't be included and what is the json?

viniciuslance commented:
> Icons is messed up, apk shouldn't be included and what is the json?

Probably a file that the android studio created. Deleted the apk and fixed the icons.

CarlosEsco commented:
Last thing the build.gradle and the Gradle.properties need to be removed. Only the build.gradle for extension is needed

viniciuslance commented:
Fixed, I copied the files before my first commit, if there's any other problem just tell me. Thanks.

LinkCable commented:
Agree with the name bullet point. Don't put BR or the language in the name of the extension.

viniciuslance commented:
Fixed the BR naming, and for the `extVersionSuffix = 0` I needed to force-push my branch to fix this.
So, I reopened the pull request. 

If there's any other problem just tell me.

alessandrojean commented:
The only thing wrong now that I see is the `ic_launcher-web.png` on the root of repository, that need to be removed, and the `src/pt/mangahost/ic_launcher-web.png` that is in the wrong place and need to be moved and renamed to `src/pt/mangahost/res/web_hi_res_512.png`. 

The `res` folder that in the root of repository need to be moved to `src/pt/mangahost/res` (and I think you replaced the default icon of the extensions without icons).

viniciuslance commented:
> Now the file `src/pt/mangahost/src/eu/kanade/tachiyomi/extension/pt/mangahost/res/web_hi_res_512.png` need to be in `src/pt/mangahost/res` and the other icons (`mipmap-hdpi`, `mipmap-mdpi`, `mipmap-xhdpi`, `mipmap-xxhdpi` and `mipmap-xxxhdpi` folders with their proper `ic_launcher.png` in each one) in the `src/pt/mangahost/res` are missing.
> 
> If you need help for generating the proper sizes of the icons, you can use [this generator](https://romannurik.github.io/AndroidAssetStudio/icons-launcher.html#foreground.type=text&foreground.text.text=M&foreground.space.trim=1&foreground.space.pad=0.3&foreColor=rgb(4%2C%20152%2C%20206)&backColor=rgb(0%2C%200%2C%200)&crop=0&backgroundShape=square&effects=none&name=ic_launcher) (the link points to an icon matching the actual MangaHost favicon, if you want, feel free to use it, just download the zip and put the `web_hi_res_512.png` in `src/pt/mangahost/res` and the contents of the `res` folder goes to `src/pt/mangahost/res`).
> 
> For example, look the `src/pt/unionmangas` folder. The `src/pt/mangahost` folder need to have the same files and structure (but for MangaHost). If you fix this, then your PR can be merged.

Thanks for the help.

alessandrojean commented:
> Thanks for the help.

You're welcome.

What about now, @CarlosEsco? I think it can be merged.
